app/controllers/api/teams.php

```diff
@@ -434,16 +464,25 @@
         $url = Template::unParseURL($url);
 
         if (!$isPrivilegedUser && !$isAppUser) { // No need of confirmation when in admin or app mode
-            $mails
-                ->setType(MAIL_TYPE_INVITATION)
-                ->setRecipient($email)
-                ->setUrl($url)
-                ->setName($name)
-                ->setLocale($locale->default)
-                ->setTeam($team)
-                ->setUser($user)
-                ->trigger()
-            ;
+            if (!empty($email)) {
```

Comment:
If both email and phone are provided, does it send both? Is that intended? I would again try to setup a priority and write documentation for it.

Reply:
Priority is ID > email > Phone, however if both  is provided, I think it's logical to send on to both?

Reply:
Sending the message on both seems a bit annoying, maybe we can try to prioritize email?

Reply:
Makes sense to me to prioritize email if both provided.
If they want to send SMS, they can invite without email and later add email (with event / function / ...)

Alternatively we could add parameter to provide array of strings where to send notificaton. `email` and `phone` would be valid options.
If we did that, in theory this endpoint could behave same for client and for server. In the end, currently server-SDK does not trigger email. With this parameter, server could decide if they want to send email or not.

Reply:
fixed